main.py

```diff
@@ -140,6 +183,23 @@ def load_image(self):
         logger.info("Loaded image size: {}", im.size)
         self.image_size = im.size
 
+    # Download file from url over http
+    def download_resouce(self, url, destination):
+        r = requests.get(url, allow_redirects=False)

```

Comment:
Can we change allow_redirects to True ? Someone could want to shorten the URL for example which would require a redirection.

Reply:
Just updated to include this